ddtrace/tracer/option.go

```diff
@@ -443,6 +443,13 @@ func WithHostname(name string) StartOption {
 	}
 }
 
+// WithLogStartup allows control over whether to print the startup log or not
+func WithLogStartup(logStartup bool) StartOption {
```

Comment:
```suggestion
// WithLogStartup determines the tracer to print additional start-up information. This is useful for debugging.
func WithLogStartup() StartOption {
```
I don't think there's any point to have the boolean, right?

Reply:
That's a good point. The default value in the config is false, but because of [this line](https://github.com/DataDog/dd-trace-go/blob/a0e288c012e4e212bab6776a1b10ad683ed315e3/ddtrace/tracer/option.go#L171) in `ddtrace/tracer/option.go` the effective default is true - if that makes sense. We could perhaps rename this function along the lines of `WithoutLogStartup` or `WithNoStartupLog` to reflect that it is suppressing the log and not enabling it.

Reply:
Ah ok, I see. You're right. Let's leave it as it is in that case. My bad.